scripts/templates/print.hpp.mako

```diff
@@ -117,49 +121,50 @@ template <> struct is_handle<${th.make_type_name(n, tags, obj)}> : std::true_typ
 %endfor
 template <typename T>
 inline constexpr bool is_handle_v = is_handle<T>::value;
-template <typename T> inline void serializePtr(std::ostream &os, const T *ptr);
-template <typename T> inline void serializeFlag(std::ostream &os, uint32_t flag);
-template <typename T> inline void serializeTagged(std::ostream &os, const void *ptr, T value, size_t size);
+template <typename T> UR_DLLEXPORT inline ${x}_result_t UR_APICALL ${x}PrintPtr(std::ostream &os, const T *ptr);
```

Comment:
Hm, do we actually need UR_DLLEXPORT? All those functions are defined in the header, right?